packages/react-devtools-shared/src/devtools/views/root.css

```diff
@@ -1,280 +1,25 @@
-  --dark-color-scheduling-profiler-priority-border: #282c34;
-  --dark-color-scheduling-profiler-user-timing: #c9cacd;
-  --dark-color-scheduling-profiler-user-timing-hover:#93959a;
-  --dark-color-scheduling-profiler-react-idle: #3d485b;
-  --dark-color-scheduling-profiler-react-idle-selected:#465269;
-  --dark-color-scheduling-profiler-react-idle-hover:#465269;
-  --dark-color-scheduling-profiler-react-render: #9fc3f3;
-  --dark-color-scheduling-profiler-react-render-selected:#64A9F5;
-  --dark-color-scheduling-profiler-react-render-hover:#2683E2;
-  --dark-color-scheduling-profiler-react-commit: #ff718e;
-  --dark-color-scheduling-profiler-react-commit-selected:#FF5277;
-  --dark-color-scheduling-profiler-react-commit-hover:#ed0030;
-  --dark-color-scheduling-profiler-react-layout-effects:#c88ff0;
-  --dark-color-scheduling-profiler-react-layout-effects-selected:#934FC1;
-  --dark-color-scheduling-profiler-react-layout-effects-hover:#601593;
-  --dark-color-scheduling-profiler-react-passive-effects:#c88ff0;
-  --dark-color-scheduling-profiler-react-passive-effects-selected:#934FC1;
-  --dark-color-scheduling-profiler-react-passive-effects-hover:#601593;
-  --dark-color-scheduling-profiler-react-schedule: #9fc3f3;
-  --dark-color-scheduling-profiler-react-schedule-hover:#2683E2;
-  --dark-color-scheduling-profiler-react-schedule-cascading:#ff718e;
-  --dark-color-scheduling-profiler-react-schedule-cascading-hover:#ed0030;
-  --dark-color-scheduling-profiler-react-suspend: #a6e59f;
-  --dark-color-scheduling-profiler-react-suspend-hover:#13bc00;
-  --dark-color-scheduling-profiler-text-color: #000000;
-  --dark-color-scheduling-profiler-react-work-border:#ffffff;
   --dark-color-scroll-thumb: #afb3b9;
   --dark-color-scroll-track: #313640;
-  --dark-color-search-match: yellow;
-  --dark-color-search-match-current: #f7923b;
-  --dark-color-selected-tree-highlight-active: rgba(23, 143, 185, 0.15);
-  --dark-color-selected-tree-highlight-inactive: rgba(255, 255, 255, 0.05);
-  --dark-color-shadow: rgba(0, 0, 0, 0.5);
-  --dark-color-tab-selected-border: #178fb9;
-  --dark-color-text: #ffffff;
-  --dark-color-text-invalid: #ff8080;
-  --dark-color-text-selected: #ffffff;
-  --dark-color-toggle-background-invalid: #fc3a4b;
-  --dark-color-toggle-background-on: #178fb9;
-  --dark-color-toggle-background-off: #777d88;
-  --dark-color-toggle-text: #ffffff;
   --dark-color-tooltip-background: rgba(255, 255, 255, 0.95);
   --dark-color-tooltip-text: #000000;
 
   /* Font smoothing */
-  --light-font-smoothing: auto;
-  --dark-font-smoothing: antialiased;
   --font-smoothing: auto;
 
   /* Compact density */
-  --compact-font-size-monospace-small: 9px;
-  --compact-font-size-monospace-normal: 11px;
-  --compact-font-size-monospace-large: 15px;
-  --compact-font-size-sans-small: 10px;
-  --compact-font-size-sans-normal: 12px;
-  --compact-font-size-sans-large: 14px;
   --compact-line-height-data: 18px;
   --compact-root-font-size: 16px;
```

Comment:
I left `root-font-size` because it is used here:

https://github.com/facebook/react/blob/9f88b5355ba21e06e0a3c4442c5199de354ed1cf/packages/react-devtools-shared/src/devtools/views/Settings/SettingsContext.js#L274-L282